managers/asset-operations-manager.js

```diff
@@ -829,29 +836,41 @@ class AssetOperationsManager {
                 blockchain,
             );
 
-            tokenAmountInWei = await this._getUpdateBidSuggestion(
-                UAL,
+            const firstAssertionId = await this.blockchainService.getAssertionIdByIndex(
+                tokenId,
+                0,
                 blockchain,
+            );
+
+            const keyword = ethers.solidityPacked(
+                ['address', 'bytes32'],
+                [contract, firstAssertionId],
+            );
+
+            const agreementId = ethers.sha256(
+                ethers.solidityPacked(
+                    ['address', 'uint256', 'bytes'],
+                    [contract, tokenId, keyword],
+                ),
+            );
+            const agreementData = await this.blockchainService.getAgreementData(
+                agreementId,
+                blockchain,
+            );
+
+            tokenAmountInWei = await this.nodeApiService.getBidSuggestion(
```

Comment:
Maybe we should add tokens in the same ratio (not bid suggestion) as asset currently has as default behaviour for this function? Just a thought